sites/footers/footers-en.md

```diff
@@ -34,23 +32,30 @@ Consult the Canada.ca Design System for guidance on which footer elements or cus
 
 ## Latest changes
 
-### Version 5.0
+### Version 6.0
 
-The following changes have been made since version 2.0 and 3.0:
+The following changes have been made since **version 2.0**:
 * Added a contextual band above the main band.
+  * GCWeb Jekyll specific: The title and links are defined globally in `_config.yml`. It is also possible to remove the contextual footer globally by removing its definition in `_config.yml`.
+  * GCWeb Jekyll specific: The title and links can be changed for specific pages in the page's `contextualFooter` property.
```

Comment:
We will need to keep the reorg change style where any GCWeb Jekyll specific was moved in another section.